packages/radix-ui-themes/src/components/base-tab-list.props.ts

```diff
@@ -1,13 +1,31 @@
 import { PropDef, colorProp, highContrastProp } from '../helpers';
 
 const sizes = ['1', '2'] as const;
+const wrapValues = ['nowrap', 'wrap', 'wrap-reverse'] as const;
+const justifyValues = ['start', 'center', 'end'] as const;
```

Comment:
Is 'between' not useful for this case? Wondering about consistency with `Flex`

Reply:
Yeah, I decided to omit `between` because it looked unreasonably bad in the context of tabs